be/src/storage/lake/tablet_manager.cpp

```diff
@@ -2,58 +2,143 @@
 
 #include "storage/lake/tablet_manager.h"
 
+#include "fmt/format.h"
+#include "fs/fs.h"
+#include "gen_cpp/AgentService_types.h"
+#include "gen_cpp/starlake.pb.h"
+#include "gutil/strings/util.h"
+#include "re2/re2.h"
+#include "service/staros_worker.h"
 #include "storage/lake/group_assigner.h"
 #include "storage/lake/metadata_iterator.h"
 #include "storage/lake/tablet.h"
 #include "storage/lake/tablet_metadata.h"
 #include "storage/lake/txn_log.h"
 #include "util/lru_cache.h"
+#include "util/raw_container.h"
+#include "util/string_parser.hpp"
 
 namespace starrocks::lake {
 
 TabletManager::TabletManager(GroupAssigner* group_assigner, int64_t cache_capacity)
         : _group_assigner(group_assigner), _metacache(new_lru_cache(cache_capacity)) {}
 
+// path $bucket:/$ServiceID/$GroupID/tbl_${TabletID}_${version}
+std::string TabletManager::tablet_meta_path(const std::string& group, int64_t tablet_id, int64_t verson) {
+    return fmt::format("{}/tbl_{:016X}_{:016X}", group, tablet_id, verson);
+}
+
+// trnslog path rule $Bucket:/$ServiceID/$GroupID/txn_${TabletID}_${TxnID}
+std::string TabletManager::txn_log_path(const std::string& group, int64_t tablet_id, int64_t txn_id) {
+    return fmt::format("{}/txn_{:016X}_{:016X}", group, tablet_id, txn_id);
+}
+
 Status TabletManager::create_tablet(const TCreateTabletReq& req) {
-    (void)req;
-    return Status::NotSupported("TabletManager::create_tablet");
+    // generate metapb
+    TabletMetadataPB tablet_metadata_pb;
+    tablet_metadata_pb.set_id(req.tablet_id);
+    tablet_metadata_pb.set_version(1);
+
+    // rowset metadata with version 1
+    auto rowset_metadata_pb = tablet_metadata_pb.add_rowsets();
+    rowset_metadata_pb->set_id(1);
+    rowset_metadata_pb->set_overlapped(false);
+    rowset_metadata_pb->set_data_size(0);
+    rowset_metadata_pb->set_num_rows(0);
+    tablet_metadata_pb.set_next_rowset_id(2);
```

Comment:
Why add a rowset?

Reply:
This is to set the default value of rowset pb.